test/e2e/suite/conformance/certificates/venafi/venafi.go

```diff
@@ -54,13 +54,15 @@ var _ = framework.ConformanceDescribe("[Feature:Issuers:Venafi:TPP] Certificates
 	provisioner := new(venafiProvisioner)
 	(&certificates.Suite{
 		Name:                "Venafi Issuer",
+		DomainSuffixType:    certificates.IstioDomainSuffixType,
```

Comment:
The `DomainSuffix` value is used in https://github.com/jetstack/cert-manager/blob/b95836421f7f3d2bbbebaa4fa3cca7128e3a97ad/test/e2e/suite/conformance/certificates/suite.go#L124-L140 to generate domains. Before, the default was `IngressDomainSuffixType`. Now, I added the `IstioDomainSuffixType` option. I opted to not use `IngressDomainSuffixType` as the default and explicitly added the `DomainSuffixType` for each `Suite`.
I'll change the value in these suites to use `IngressDomainSuffixType` instead of `IstioDomainSuffixType`.
Maybe, additionally, I should use `IngressDomainSuffixType` as the default and not require the value to be set explicitly? What do you think @wallrj?